components/Layout/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```

Comment:
In the newest versions of React, you don't need to import React ,maybe you can remove this import.